### DISCUSSION TITLE
ProjectDiscovery Final Cloud

### DISCUSSION THREAD

**Discussion by waall** [projectdiscovery/nuclei-templates#7781] [General]
I am writing here on behalf of some inegrants, all the tools of the DP, in the end interconnect being a nice security solution.... What worries me is that the voluntary labor in the end will be converted into a unique tool in the cloud that will be sold, that is, are not they abusing the community for their own benefit? Note that this is only a speculation and not an insunation, happy hacking everyone!

**Comment by olearycrew** [comment]
Thanks for your feedback @waall !

Have you seen our announcement about the cloud product? We're hoping that it clearly states our intentions, but I'd love your feedback on what we could add to it to make the community comfortable: https://blog.projectdiscovery.io/announcing-nuclei-cloud/
